cli.js

```diff
@@ -77,18 +101,23 @@ teti({ url, num, notify, runner, custom }).then(output => {
         'σ²',
         'MAD'
     ].map(h => chalk.cyan(h));
-
-    const t = table([headings].concat(output.map(o => {
-        return [
-            chalk.blue(o.name),
-            chalk.yellow(o.metrics.median),
-            chalk.yellow(o.metrics.mean),
-            chalk.yellow(o.metrics.p95),
-            chalk.yellow(o.metrics.variance),
-            chalk.yellow(o.metrics.mad)
-        ];
-    })), { align: [ 'r', 'r', 'r', 'r', 'r', 'r' ] });
-
-    console.log(`\nResults for ${chalk.bgMagenta(url)} based on ${chalk.bgMagenta(num)} requests:\n`);
-    console.log(`${t}\n`);
-});
+    const rows = output.map(o => [
+        chalk.blue(o.name),
+        chalk.yellow(o.metrics.median),
+        chalk.yellow(o.metrics.mean),
+        chalk.yellow(o.metrics.p95),
+        chalk.yellow(o.metrics.variance),
+        chalk.yellow(o.metrics.mad)
+    ]);
+
+    if (outputFormat === 'table') {
+        const t = table([headings, ...rows], {
+            align: Array.from(headings, () => 'r')
+        });
+
+        console.log(`\nResults for ${chalk.bgMagenta(url)} based on ${chalk.bgMagenta(number)} requests:\n`);
+        console.log(`${t}\n`);
+    } else {
+        console.log(csv([headings, ...rows]));
```

Comment:
console.table?

Reply:
That's a great idea! But I'd recommend to add it as an option, for two reasons:
 * it's not supported in each node.js version (only v10+ I believe)
 * it's not really equal to printing both "formatted table" and CSV - we have mandatory `(index)` column plus a lot of characters for borders that are meaningless but you have to copy them when copying the output
```
$ console.table(['foo', 'bar'], ['cat', 1], ['dog', 1])
┌─────────┬───┬─────┐
│ (index) │ 1 │ cat │
├─────────┼───┼─────┤
│    0    │   │     │
│    1    │   │     │
└─────────┴───┴─────┘
```

So, maybe: `pretty`, `table`, `csv`?